site/src/docs/components/highlight/tabs.mdx

```diff
@@ -0,0 +1,30 @@
+---
+slug: '/components/highlight/tabs'
+title: 'Highlight'
+---
+
+import { StatusLabel } from 'hds-react';
+
+import LeadParagraph from '../../../components/LeadParagraph';
+import Layout from '../../../components/layout';
+import PageTabs from '../../../components/PageTabs';
+
+# Highlight
+
+<StatusLabel type="alert" variant="rounded">Beta</StatusLabel>
```

Comment:
This is missing the Accessible StatusLabel and tooltip. Copy the implementation from another component's docs.

Reply:
The tooltip is missing too. If you look at for example `site/src/docs/components/accordion/tabs.mdx`, there's the StatuslabelTooltip component after these, and all of them are wrapped inside a `<div class="status-label-description">`.